package.json

```diff
@@ -58,9 +61,12 @@
     "prettier": "^3.0.0",
     "prettier-plugin-tailwindcss": "^0.5.4",
     "tailwindcss": "^3.3.3",
-    "typescript": "^5.1.6"
+    "typescript": "^5.1.6",
+    "bun-types": "latest"
```

Comment:
Why did you add this?

Reply:
propably added itself when adding some deps. Unused and will be deleted.